data/transliterations/cz2ua.ts

```diff
@@ -1,4 +1,4 @@
-export const cz2ua:[string,string][] = [
+export const cz2ua: [string, string][] = [
```

Comment:
Add comment at the beginning indicating that here we are using the code of someone (also MIT licence, but indicate the source)

// Based on MIT licensed code of Vaclav Sistek https://github.com/vsistek/ua-translit-cz